pkg/limatype/lima_instance.go

```diff
@@ -47,6 +48,8 @@ type Instance struct {
 	Protected       bool              `json:"protected"`
 	LimaVersion     string            `json:"limaVersion"`
 	Param           map[string]string `json:"param,omitempty"`
+	// Guest IP address directly accessible from the host.
+	GuestIP net.IP `json:"guestIP,omitempty"`
```

Comment:
Can't we reuse `SSHAddress` in L47?

Currently this still refers to "127.0.0.1" even in direct IP mode